calico/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -51,15 +51,11 @@ eBPF (or "extended Berkeley Packet Filter"), is a technology that allows safe mi
 
 - IPv6
 
-:::note
-This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.
-:::
-
 Limitations:
 
-  - Dual stack is not supported. However, ipv4 traffic is allowed into hosts (not workloads) because many managed Kubernetes environments have ipv4-based control plane.
-  - `doNotTrack` policies (xdp) are not supported
```

Comment:
Did you intend to remove this? Is this a ticketed item?

Reply:
that is correct, we have v6 support for this now

Reply:
Is line 61 related to IPv6?

Reply:
This whole block is ipv6, the feature preview note above made it a bit more difficult to see that. I noticed that. But that is all going away.